src/components/SvgMousePosition/enhance.js

```diff
@@ -0,0 +1,16 @@
+import { noop } from 'lodash';
+import {
+  compose,
+  withProps,
+  defaultProps,
+} from 'recompose';
+import { once } from 'ramda';
```

Comment:
I know it's not related to the PR but: is this really need to have both ramda and lodash? They should be used interchangeably and if you need something from `lodash` that `ramda` doesn't have you can use the specific packages, like this one: https://www.npmjs.com/package/lodash.throttle